index.d.ts

```diff
@@ -35,59 +35,113 @@ declare namespace tempy {
 		*/
 		readonly prefix?: string;
 	};
+
+	/**
+	The temporary path created by the function.
+	*/
+	type TaskCallback = (tempPath: string) => void;
```

Comment:
In the JS code, the callback is awaited. We should reflect that in the types here and in the readme. Also with a note that the callback can be async.

Reply:
Done!